### ISSUE TITLE
Starring projects

### ISSUE THREAD

**Issue by gratimax** [SpongePowered/Ore#44]
- [ ] star projects
- [ ] view people who star project
- [ ] view stars of somebody

**Comment by gratimax** [comment]
Giving this to @d4rkfly3r after some chat in Slack

**Comment by JBYoshi** [comment]
Question: Should users be able to star their own projects?

**Comment by gratimax** [comment]
Yes, and users should have their own projects starred by default.
